runtime-builder/build.sh

```diff
@@ -26,7 +26,7 @@ usage() { echo "Usage: $0 <project> <go_version>"; exit 1; }
 debian_digest()
 {
   # We sometimes have issue with 'describe', use the --log-http flag to get more info if it fails.
-  local digest="$(gcloud container images describe gcr.io/google-appengine/debian8:latest | \
+  local digest="$(gcloud alpha container images describe gcr.io/google-appengine/debian8:latest | \
```

Comment:
No, this is to record which version we use for the builder.

We can also record distroless/base digest, that will be in go-build.sh, when building the output image.